src/components/Hooks/useAuth.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { useContext } from 'react';
+import AuthContext from '../Store/auth-context';
+
+export const useAuth = () => {
+  return useContext(AuthContext);
```

Comment:
this is not what I mean, check screenshots for example, and hooks is not what returns context, please check these things with levon
<img width="864" alt="Screen Shot 2022-03-09 at 19 10 22" src="https://user-images.githubusercontent.com/55436142/157469941-39f8a4fa-b66e-44b6-a92f-8435b0ad8288.png">

<img width="723" alt="Screen Shot 2022-03-09 at 19 11 00" src="https://user-images.githubusercontent.com/55436142/157470030-71930750-d12a-4248-9de7-e8386e08a842.png">